app/frontend/src/cljs/teet/project/project_navigator_view.cljs

```diff
@@ -433,7 +433,9 @@
         (when (and
                 (common-controller/feature-enabled? :vektorio)
                 (some? (:vektorio/project-id project)))
-          [common/vektorio-link {:href (common-controller/query-url :vektorio/instant-login project)
+          [common/vektorio-link {:href (common-controller/query-url
+                                         :vektorio/instant-login {:db/id (:db/id project)
+                                                                  :vektorio/project-id (:vektorio/project-id project)})
                                  :target "_blank"}
            (str (tr [:project :bim-models]) (:vektorio/project-id project))])
```

Comment:
There should probably be a space between the translation and the vektorio project-id.

Reply:
There is no space between THK and project id, but I agree with a space it looks better, just different from THK link